libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt

```diff
@@ -28,56 +28,75 @@ internal object LoggerProvider {
 /**
  * Alias of [LoggerProvider.logger]#v
  */
-fun logV(tag: String? = null, msg: String) {
+fun logV(msg: String, category: String? = null) {
+    val message = createMessage(msg, category)
     // There's no com.mapbox.common.Logger.v available - using Logger.d instead
-    com.mapbox.common.Logger.d(tag, msg)
+    com.mapbox.common.Logger.d(TAG, message)
 }
```

Comment:
This change (and in all other functions) is not needed, is it?

Don't we just need the change that creates a message in `NavigationLogBackend` and we could leave the rest as it was?

Reply:
Well it depends if we want to prepend the class name or not. Implemented both https://github.com/mapbox/mapbox-navigation-android/pull/5604/commits/4518c17fde5fa9c8e645aaba2d7c6b9b31a1f49a https://github.com/mapbox/mapbox-navigation-android/pull/5604/commits/42b089d810a515f6d35f9b25a89c1f142c1145aa

In any case, `category` is always `Mapbox` even setting it here to `nav-sdk` https://github.com/mapbox/mapbox-navigation-android/blob/42b089d810a515f6d35f9b25a89c1f142c1145aa/libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt#L22 🤔 

It seems it's overridden somehow or above is not working. @mr1sunshine any ideas?

Reply:
This call `com.mapbox.common.Logger.addCategoryFilter(CATEGORY)` will filter out all the logs with provided category. So with your change, you will filter out all the logs with `nav-sdk` category.

Reply:
I think the confusion was because `common` is using "tag" as "category" behind the scenes 😬 

All good now 👍